### PR TITLE
fix(ui): update copy

### LINKED ISSUE #877: Copy updates
Location: In the start a new project modal, in the "Create a new repository" section.
- [x] Copy: `A new repository will be created on your machine inside a directory and named after the project name.`

---

Location: In the git helper in the new project modal.
- [x] Copy: `To publish code to Radicle, you need to add this to your shell configuration file. Not sure how? Read more`

---

Location: new project modal, continue with an existing repo section.
- [x] Add Copy: `This will publish the chosen repository to the Radicle network.`

---

Location: Global
- [ ] Replace all instances of "Track" with "Follow" (Make sure to get all instances of "Tracking" or "Tracked".)

---

Location: onboarding passphrase step
- [x] `You can’t recover your account with it` should be `typo-text-bold`

---

Location: Settings
- [x] change "Seed addresses" to "Network"
- [x] change description text to `Enter seed addresses that you’d like to subscribe to here. Projects replicated by seeds will appear in the Discover page. <a>Learn more about seeds</a>`
- [x] remove "Learn more about seeds" in the top row of the section.

---

Location: onboarding, name step
- [x] change `name` to `display name`

---

Location: new project creation
- [x] "This directory should be empty" error should change to "Please choose a directory where Git is not installed."

### PR DESCRIPTION
**Pull request by juliendonck**
closes #877 

Only changed the wording of `Tracking` to `following`, didn't rename the variables and components (yet).

### CODE REVIEW

**Review by rudolfs — APPROVED** [review]

File: ui/Screen/Onboarding/EnterName.svelte [hunk 1]
```diff
@@ -68,9 +68,9 @@
       what should we call you?
     </h1>
     <p>
-      You’ll need a name to interact on Radicle. This isn’t unique across the
-      platform, but it helps others recognize you a little easier. You can
-      change it in your profile at any time.
+      You’ll need a display name to interact on Radicle. This isn’t unique
+      across the platform, but it helps others recognize you a little easier.
+      You can change it in your profile at any time.
```
Comment (cloudhead) [inline ui/Screen/Onboarding/EnterName.svelte:73]: You can't change it yet in the B1.

```suggestion
```

File: ui/Screen/Onboarding/EnterName.svelte [hunk 2]
```diff
@@ -68,9 +68,9 @@
       what should we call you?
     </h1>
     <p>
-      You’ll need a name to interact on Radicle. This isn’t unique across the
-      platform, but it helps others recognize you a little easier. You can
-      change it in your profile at any time.
+      You’ll need a display name to interact on Radicle. This isn’t unique
+      across the platform, but it helps others recognize you a little easier.
```
Comment (cloudhead) [inline ui/Screen/Onboarding/EnterName.svelte:72]:
```suggestion
      across the network, but it helps others recognize you a little easier.
```

File: ui/Screen/Settings.svelte [hunk 3]
```diff
@@ -119,28 +119,31 @@
...
             Seeds help you see more projects and people on the network
           </p>
+          <!-- TODO(julien): link to actual docs abt seeds -->
           <p
             style="color: var(--color-foreground-level-6); margin-bottom: 24px;">
-            Have some seed addresses you’d like to join? Enter them here and new
-            projects from the seeds you’re subscribed to will appear in the
-            Discover page.
+            Enter seed addresses that you’d like to subscribe to here. Projects
+            replicated by seeds will appear in the Discover page. <a
```
Comment (cloudhead) [inline ui/Screen/Settings.svelte:133]: No discover page in B1

File: ui/Screen/Settings.svelte [hunk 4]
```diff
@@ -119,28 +119,31 @@
...
+            </a>
           </p>
         </div>
         <div class="action typo-mono">
           <Input.Textarea
             style="flex: 1; height: 6rem;"
             bind:value={seedInputValue}
             on:change={updateSeeds}
-            placeholder="Enter seeds here" />
+            placeholder="Enter seed addresses here" />
```
Comment (cloudhead) [inline ui/Screen/Settings.svelte:146]: "Seeds" would be more accurate, since a seed is peerid@address

Comment (juliendonck) [reply ui/Screen/Settings.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/892/commits/475ee1ea2246c55555e629fb29d0e1db87289a89

File: ui/Screen/Settings.svelte (hunk 3 above)
Comment (juliendonck) [reply ui/Screen/Settings.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/892/commits/475ee1ea2246c55555e629fb29d0e1db87289a89

File: ui/Screen/Onboarding/EnterName.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/Screen/Onboarding/EnterName.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/892/commits/475ee1ea2246c55555e629fb29d0e1db87289a89

File: ui/Screen/Onboarding/EnterName.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/Screen/Onboarding/EnterName.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/892/commits/475ee1ea2246c55555e629fb29d0e1db87289a89

**Review by cloudhead — APPROVED** [review]

File: ui/Screen/Settings.svelte [hunk 5]
```diff
@@ -119,20 +119,25 @@
...
           </p>
+          <!-- TODO(julien): link to actual docs abt seeds -->
           <p
             style="color: var(--color-foreground-level-6); margin-bottom: 24px;">
-            Have some seed addresses you’d like to join? Enter them here and new
-            projects from the seeds you’re subscribed to will appear in the
-            Discover page.
+            Enter seed addresses that you’d like to subscribe to here.
+            <!-- Projects
+            replicated by seeds will appear in the Discover page. -->
```
Comment (rudolfs) [inline ui/Screen/Settings.svelte:134]: Can we just delete this?

**Review by rudolfs — APPROVED** [review]

**Review by sarahscott — APPROVED** [review]

File: ui/Screen/Settings.svelte (hunk 5 above)
Comment (juliendonck) [reply ui/Screen/Settings.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/892/commits/1be04550dac4f70f85861b9ae8189bda898ca978

**Review by rudolfs — APPROVED** [review]

**Review by abbey-titcomb — APPROVED** [review]
lgtm!

**Review by cloudhead — APPROVED** [review]

**Pull request merged**